gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/util/FormatHelper.java

```diff
@@ -259,6 +259,16 @@ public static String formatDateTime(final Date date) {
 		return df.format(date);
 	}
 
+	/**
+	 * Strips out line breaks
+	 *
+	 * @param s String to abbreviate
+	 * @return string without line breaks
+	 */
+	public static String stripLineBreaks(final String s) {
+		return s.replaceAll("\\r\\n|\\r|\\n", "");
```

Comment:
It's a shame Commons Lang doesn't have a method to do this - only trim.

Reply:
Agreed. I was surprised not to find an existing method ....